Missing include in includes/register_params.hpp for successfully compiling version 1.2.0

```
Compiling: src/board.cpp -> build/board.o
         Compile time: 5
Compiling: src/endgames.cpp -> build/endgames.o
         Compile time: 3
Compiling: src/eval.cpp -> build/eval.o
         Compile time: 4
Compiling: src/hashing.cpp -> build/hashing.o
         Compile time: 15
Compiling: src/hashtable.cpp -> build/hashtable.o
         Compile time: 2
Compiling: src/history.cpp -> build/history.o
         Compile time: 1
Compiling: src/move_queue.cpp -> build/move_queue.o
         Compile time: 8
Compiling: src/movegen.cpp -> build/movegen.o
         Compile time: 8
Compiling: src/parse_format.cpp -> build/parse_format.o
         Compile time: 8
Compiling: src/pst.cpp -> build/pst.o
         Compile time: 5
Compiling: src/read_game.cpp -> build/read_game.o
         Compile time: 5
```
```
Compiling: src/register_params.cpp -> build/register_params.o
In file included from src/register_params.cpp:1:
includes/register_params.hpp:36:9: error: 'uint64_t' does not name a type
   36 |         uint64_t priority;
      |         ^~~~~~~~
includes/register_params.hpp:8:1: note: 'uint64_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
    7 | # include <optional>
  +++ |+#include <cstdint>
    8 |
includes/register_params.hpp: In function 'constexpr bool operator<(TweakWithPriority, TweakWithPriority)':
includes/register_params.hpp:41:18: error: 'const struct TweakWithPriority' has no member named 'priority'
   41 |         return x.priority < y.priority;
      |                  ^~~~~~~~
includes/register_params.hpp:41:31: error: 'const struct TweakWithPriority' has no member named 'priority'
   41 |         return x.priority < y.priority;
      |                               ^~~~~~~~
src/register_params.cpp: In function 'TweakQueue initialize_queue(int)':
src/register_params.cpp:59:77: error: cannot convert 'ProposedTweak' to 'size_t' {aka 'long long unsigned int'} in initialization
   59 |                         queue.push(TweakWithPriority{0xFFFF'FFFF'FFFF'FFFF, ProposedTweak{i, j, cur_val > 0 ? -starting_mod : starting_mod}});
      |                                                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                                             |
      |                                                                             ProposedTweak
make[1]: *** [Makefile:191: build/register_params.o] Error 1
make: *** [Makefile:106: release] Error 2
```
After adding the missing `#include <cstdint>` the compilation went fine.

Guenther (RWBC)

--- a/includes/register_params.hpp
+++ b/includes/register_params.hpp
@@ -4,6 +4,7 @@
 # include <array>
 # include <tuple>
 # include <queue>
+# include <cstdint>
 # include <optional>
 
 void register_param(std::string param_name, int *param_ref, size_t length);

